modules/terraform-aws-alternat/versions.tf

```
@@ -0,0 +1,10 @@
+terraform {
+  required_version = ">= 1.0"
+
+  required_providers {
+    aws = {
+      source  = "hashicorp/aws"
+      version = ">= 5.20"
```

We currently use alternat with both v4 and v5 of the AWS provider, so I think it'd be safe to do:

```suggestion
      version = ">= 4"
```